otelcollector/configmapparser/default-prom-configs/clusterautoscalerDefault.yml

```diff
@@ -0,0 +1,28 @@
+  scrape_configs:
+  - job_name: cluster-autoscaler
+    scrape_interval: $$SCRAPE_INTERVAL$$
+    follow_redirects: false
+    kubernetes_sd_configs:
+      - role: pod
+        namespaces:
+          names:
+            - $$POD_NAMESPACE$$
+    relabel_configs:
+      - source_labels: [ __meta_kubernetes_pod_label_app, __meta_kubernetes_pod_container_name ]
+        action: keep
+        regex: cluster-autoscaler;cluster-autoscaler
+      - source_labels: [ __meta_kubernetes_pod_annotation_aks_prometheus_io_scrape ]
+        action: keep
+        regex: true
+      - source_labels: [ __meta_kubernetes_pod_annotation_aks_prometheus_io_path ]
+        action: replace
+        target_label: __metrics_path__
+        regex: (.+)
+      - source_labels: [ __address__, __meta_kubernetes_pod_annotation_aks_prometheus_io_port ]
+        action: replace
+        separator: ":"
```

Comment:
we need to replace the address with "regex: (.+?)(\:\d+)?" but keeping it like this while we test in standalone